parts/k8s/artifacts/1.5/kuberneteskubelet.service

```diff
@@ -1,45 +0,0 @@
-[Unit]
```

Comment:
Removed the 1.5 version since k8s 1.5 was deprecated in #2394 and it's not in use anymore